LameDLLWrap/LibMp3Lame.cs

```diff
@@ -577,24 +577,55 @@ public void ID3SetPad(int nBytes)
 		/// <param name="title">Value to set</param>
 		public void ID3SetTitle(string title)
 		{
-			NativeMethods.id3tag_set_title(context, title);
+			// XXX: Should we use Encoding.ASCII instead of UTF8?
```

Comment:
Encoding.ASCII replaces all non-ASCII characters with a '?' character. A proper Latin1 detection would need a round-trip convert and compare like this:

```
static bool IsLatin1String(string s) => s == Encoding.Latin1.GetString(Encoding.Latin1.GetBytes(s));
```

Not necessarily the most efficient method, but it works.